src/openklant/tests/test_migrate.py

```diff
@@ -689,3 +693,78 @@ def test_rsin(self):
         )
 
         self.assertEqual(output, [f"{self.live_server_url}{partij_url}"])
+
+    def test_telefoonnummer(self):
```

Comment:
Just to be sure, could you also add this test for a Klant with type `niet_natuurlijk_persoon`?